Detectors/TRD/qc/include/TRDQC/Tracking.h

```diff
@@ -118,13 +123,21 @@ class Tracking
   // Make output accessible to DPL processor
   std::vector<TrackQC>& getTrackQC() { return mTrackQC; }
 
+  // Set the local gain factors with values from the ccdb
+  void setLocalGainFactors(o2::framework::ProcessingContext& pc)
```

Comment:
```suggestion
  void setLocalGainFactors(const o2::trd::LocalGainFactor* padGain)
```
Then you can drop the Framework includes above

Reply:
Ok